test/system/system-test.ts

```diff
@@ -6,22 +6,89 @@
 /**
  * Performs a series of system tests.
  */
-import { spawn, fork, exec } from "child_process";
+import { ChildProcessWithoutNullStreams, spawn } from "child_process";
+import { ok, strictEqual } from "assert";
+import { dinosaursShowTabs as expectedDinosaursShowTabs } from "../expected/index";
+import { fruits as expectedFruits } from "../expected/index";
+import { fruitsNumber as expectedFruitsNumber } from "../expected/index";
+import { fruitsShowEnds as expectedFruitShowEnds } from "../expected/index";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5179809)